### PR TITLE
Fixed 1875: Sort order fixed on dragndrop

### PR DESCRIPTION
**Pull request by martinb14**
http://platform.dandigbib.org/issues/1875
And a small unrelated bug

### CODE REVIEW

**Comment by cableman** [comment]
Please fix commit messages to follow:

> [issue number]: [short summary of change]. The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

File: modules/p2/ding_list/plugins/content_types/list_menu.inc [hunk 1]
```diff
@@ -52,7 +52,7 @@ function ding_list_list_menu_content_type_render($subtype, $conf, $panel_args, $
     if (ding_list_user_has_access($list, 'edit', $user)) {
       $block->content['#links']['edit'] = array(
         'href' => $uri['path'] . '/edit',
-        'title' => t('Edit list'),
+        'title' => t('Edit list title and note'),
```
Comment (cableman) [inline modules/p2/ding_list/plugins/content_types/list_menu.inc]: I'm not sure how this relates to the issue.

Comment (cableman) [inline modules/p2/ding_list/plugins/content_types/list_menu.inc]: Sure read the issue again :-) Please dis-regard this comment

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

**Pull request merged**